arm-powerbiembedded/2016-01-29/swagger/powerbiembedded.json

```diff
@@ -0,0 +1,1041 @@
+{
+    "swagger": "2.0",
+    "host": "management.azure.com",
+    "info": {
+        "version": "2016-01-29",
+        "title": "Power BI Embedded Management Client",
+        "description": "Azure Resource Provider for Power BI Embedded"
+    },
+    "schemes": [
+        "https",
+        "http"
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "paths": {
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.PowerBI/workspaceCollections/{workspaceCollectionName}": {
+            "get": {
+                "tags": [
+                    "WorkspaceCollections"
+                ],
+                "operationId": "WorkspaceCollections_getByName",
+                "description": "The **Get Power BI Workspace Collection** operation retrieves an existing Power BI Workspace Collection.",
```

Comment:
- Why do you have `**` in the description ?
- The part between double stars is confusing. How about `Retrieves an existing Power BI Workspace Collection`
- What is the difference between WorkspaceCollection**s** and WorkspaceCollection in the part before underscore in the operationId. To avoid the confusion and the conflict between operationgroup and model definition can we just have WorkspaceCollection**s** as the operationGroup in the operationId?